packages/elcollectooorr/skills/elcollectooorr_abci/fsm_specification.yaml

```diff
@@ -1,40 +1,192 @@
 alphabet_in:
+- BASKET_DONE
+- BASKET_PERMISSION
+- CHECK_HISTORY
+- CHECK_LATE_ARRIVING_MESSAGE
+- CHECK_TIMEOUT
```

Comment:
Why has this changed in a bump PR?

Reply:
I think we might be missing some CI checks then. I'll approve, but this should be addressed.

Reply:
Opened https://github.com/valory-xyz/agent-academy-1/issues/199#issue-1642572261.

cc @0xArdi